site/src/css/custom.css

```
@@ -7,6 +7,15 @@
 @import url('https://fonts.googleapis.com/css2?family=Overpass:wght@200&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Overpass+Mono:wght@300&display=swap');
 
+/* latin */
+@font-face {
+  font-family: 'Overpass-Bold';
```

```suggestion
  font-family: 'Overpass Bold';
```
Could we rename this without the dash, only to keep it consistent with the "Overpass Mono" which we have defined in here.